src/sentry/web/frontend/oauth_revoke.py

```diff
@@ -0,0 +1,155 @@
+import logging
+from hashlib import sha256
+from typing import Optional
+
+from django.db.models import Q
+from django.http import HttpRequest, HttpResponse
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import never_cache
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic.base import View
+
+from sentry.models import ApiApplication, ApiApplicationStatus, ApiToken
+from sentry.utils import json
+
+logger = logging.getLogger("sentry.api.oauth_revoke")
+
+
+class OAuthRevokeView(View):
+    """
+    OAuth 2.0 token revoke endpoint per RFC 7009
+    https://www.rfc-editor.org/rfc/rfc7009
+
+    Clients can provide either the access_token or refresh_token in the request.
+
+    When revoking the token, the associated access_token or refresh_token is also
```

Comment:
Does this mean that if I pass in an `access_token`, then any `refresh_token` attached to it would be revoked as well (and vice versa)?